mgmt/RecordsConfig.cc

```diff
@@ -1976,6 +1976,8 @@ static const RecordElement RecordsConfig[] =
   //############
   {RECT_CONFIG, "proxy.config.http2.enabled", RECD_INT, "0", RECU_RESTART_TM, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http2.stream_priority_enabled", RECD_INT, "0", RECU_RESTART_TM, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
I agree with both of them should be `RECU_DYNAMIC`. ( I just copied `proxy.config.http2.enabled` :p )